rdrf/rdrf/template_loaders/translation.py

```diff
@@ -0,0 +1,46 @@
+import logging
```

Comment:
What about changing the name to something unrelated to i18? I see this as a rdrf specific template loader - nothing more. Along that same thread, I don't see why PRELUDE is needed. That doesn't occur to me to be a concern of the template loader.